internal/backends/winit/renderer/sw.rs

```diff
@@ -51,16 +50,15 @@ impl<const BUFFER_COUNT: usize> super::WinitCompatibleRenderer for SoftwareRende
                 type TargetPixel = i_slint_core::swrenderer::Rgb565Pixel;
                 fn process_line(
                     &mut self,
-                    line: PhysicalLength,
-                    range: core::ops::Range<PhysicalLength>,
+                    line: usize,
+                    range: core::ops::Range<usize>,
                     render_fn: impl FnOnce(&mut [Self::TargetPixel]),
                 ) {
-                    let len = (range.end.get() - range.start.get()) as usize;
-                    let line_begin = line.get() as usize * self.line.len();
+                    let len = range.end - range.start;
+                    let line_begin = line * self.line.len();
                     let sub = &mut self.line[..len];
                     render_fn(sub);
-                    for (dst, src) in self.buffer[line_begin + (range.start.get() as usize)
-                        ..line_begin + (range.end.get() as usize)]
+                    for (dst, src) in self.buffer[line_begin + range.start..line_begin + range.end]
```

Comment:
Not sure about this one:

```suggestion
                    for (dst, src) in self.buffer[line_begin..][range]
```